mkdocs.yml

```
@@ -62,6 +62,7 @@ nav:
     - Custom labels and annotations: usage/custom_labels_annotations.md
   - Reference:
     - ArgoCD: reference/argocd.md
+    - ApplicationSet Policies: reference/applicationSet.md
```

```suggestion
    - ApplicationSet:
        - Policies: reference/applicationSet.md
```
Can we create a section under ApplicationSet so that in future all applicationset related things can be added here?